src/utils.ts

```diff
@@ -102,14 +102,14 @@ export namespace Utils {
    * @param {string} challengeTx SEP0010 challenge transaction in base64.
    * @param {string} serverAccountID The server's stellar account (public key).
    * @param {string} networkPassphrase The network passphrase, e.g.: 'Test SDF Network ; September 2015'.
-   * @param {string} [homeDomain=undefined] The home domain that should be included in the Manage Data operation's string key.
+   * @param {string} [homeDomain=undefined] The field is reserved for future use and not used.
    * @returns {Transaction|string} The actual submited transaction and the stellar public key (master key) used to sign the Manage Data operation.
    */
   export function readChallengeTx(
     challengeTx: string,
     serverAccountID: string,
     networkPassphrase: string,
-    homeDomain?: string,
+    _homeDomain?: string,
```

Comment:
is the `_` syntax to signal it isn't used?

Reply:
@leighmcculloch why not just remove the value? Is it because we don't want to add a breaking change?

Reply:
@JakeUrban The proceeding underscore on params is a common pattern to indicate the variable is used, and it happens to quieten the TypeScript linter which was erroring on the variable.

@abuiles The main reason I kept the variable is because we haven't decided what we want to do with the home domain field, but also so this isn't breaking. In the next PR for SEP-10 v3.0.0 (see #582) I add another variable here called the server hostname. That'll need to be a breaking change so maybe let's discuss on that PR what to do with home domain specifically in the JS SDK's API.

Reply:
The field will come back in the next version I think.